projects/court-case-and-delius/src/main/kotlin/uk/gov/justice/digital/hmpps/api/proxy/CommunityApiController.kt

```diff
@@ -57,7 +59,68 @@ class CommunityApiController(
             log.error("Exception thrown when calling ${communityApiUrl + request.requestURI}. community-api returned ${ex.message}")
             ResponseEntity.status(ex.statusCode)
                 .headers(ex.responseHeaders)
-                .body(ex.responseBodyAsString);
+                .body(ex.responseBodyAsString)
+        }
+    }
+
+    @PostMapping("/compare")
+    fun compare(@RequestBody compare: Compare, request: HttpServletRequest): CompareReport {
+        val headers = request.headerNames.asSequence().associateWith { request.getHeader(it) }
+            .filter { it.key.lowercase() == "Authorization".lowercase() }.toMutableMap()
+
+        headers.forEach { log.info("Header name: ${it.key}") }
+
+        val uri = try {
+            Uri.valueOf(compare.uri)
+        } catch (ex: Exception) {
+            return compare.toReport("This endpoint is not configured")
```

Comment:
Is it worth just throwing on these exceptions instead of catching and returning a 200?  Would simplify the code, and means we can see them in Sentry etc.

Reply:
Removed exception handling